### ISSUE TITLE
Support v2.1 new VRP algorithms (pgr_vrpOneDepot/pgr_vrppdtw)

### ISSUE THREAD

**Issue by sanak** [pgRouting/pgRoutingLayer#2]
pgRouting 2.1.0 release will come soon, so consider to support new VRP algorithm as soon as possible.
http://lists.osgeo.org/pipermail/pgrouting-dev/2015-July/001553.html

**Comment by cvvergara** [comment]
Also, there are new signatures for pgr_dijkstra, pgr_ksp and pgr_drivingDistance

**Comment by sanak** [comment]
@cvvergara 
Oh, okay.
Thanks for information!

**Comment by dkastl** [comment]
One question might be how the distance matrix should be computed.

**Comment by cvvergara** [comment]
@dkastl 
There is this function [pgr-points-to-dmatrix](http://docs.pgrouting.org/v2.1.0-alpha/src/common/doc/convenience/points_to_dmatrix.html#pgr-points-to-dmatrix)

So if you do a:
```
select * from pgr_djkstra( <query> , array[2,3,4,5], array[2,3,4,5]);
```
and then call that function you can get a martix.
(but I really don't understnad the documentation of pgr-points-to-dmatrix)

**Comment by woodbri** [comment]
You probably want something like this:
```
select * from pgr_pointsToDMatrix(
    select array_agg(geom)
      from nodes
      where id = ANY(array[2,3,4,5])
      order by id -- force an know order for the id's
);
```
or if you want to use the new pgr_dijkstra() it is a little more complicated to create an dmatrix because it needs to be a double[][] object so you have to do an array_agg for each row and then array_agg again to assemble the rows into a matrix.

**Comment by cvvergara** [comment]
you can use  pgr_vidstodmatrix?

**Comment by woodbri** [comment]
Yes, this is correct. I built a commplete set of tools for working with pgr_TSP.

**Comment by sanak** [comment]
Okay, thanks for more information.
I will check those out, then write supported functions
to the wiki (https://github.com/pgRouting/pgRoutingLayer/wiki).

**Comment by cvvergara** [comment]
From the title of the issue, I think the conversation diverted to other topics. So I am retaking the title of the topic. VRP functions.

Even that they exist on pgRouting, they are going to be fixed.
I would like to add them to the plug in when they are fixed.

**Comment by cvvergara** [comment]
The algorithms are still experimental, and many changes are going on.

**Comment by cvvergara** [comment]
vrp functions are experimental and removed from v3.0.0 of the plugin as they can still change
as they wont be fixed in here (v3.0.0 nothing to fix as they don't exist)
Closing
